src/components/TextButton/TextButton.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import TextButton from './TextButton';
+
+export default {
+  title: 'Components/TextButton',
+  component: TextButton
+};
+
+export const TxtButton = () => <TextButton content="Text button" />;
```

Comment:
Change the name of TxtButton to `TextButtonWithContent`

Reply:
I've made the above changes but get the following error:

ERROR in
src\components\TextButton\TextButton.jsx
  Line 6:19:  The button type attribute must be specified by a static string or
a trivial ternary expression  react/button-has-type

Obviously some issue with 'type' . For the PropTypes I tried:
type: PropTypes.oneOf(['button', 'submit', 'reset']),

Also I have another build error if I delete the node_modules folder how do I reinstall with yarn?

Reply:
I think that's because you need to pass the type prop to the button in storybook.

`<TextButton content="Text button" type="button" />;`

Reply:
The type prop was already being passed. I tried changing line 14 of TextButton.jsx:

![image](https://user-images.githubusercontent.com/51328026/104725665-4dc60c80-572a-11eb-8c85-023e98742e7c.png)


from  **type: PropTypes.string.isRequired**, to **type: PropTypes.oneOf(['button', 'submit', 'reset']),**
Could be a syntax issue?

I'm still getting a build error when trying to run storybook. I've tried deleting node_modules and running yarn install but it didn't fix it

![image](https://user-images.githubusercontent.com/51328026/104725112-844f5780-5729-11eb-86a6-ceb0bea57744.png)

I came across this where they are reporting a similar issue. Not sure I follow it but may be of help?
https://stackoverflow.com/questions/61291586/webpack-htmlwebpackplugin-injection-of-chunks-into-custom-template-doesnt-wo

Reply:
Can you push your changes up so I can look at it locally please?

Reply:
Also, when your adding code snippets try to add it like this:

```
code snippet goes here
```

Makes it easier for me to copy and paste stuff. 😄

Reply:
You highlight the text then click the brackets above the markdown field 👍